Cubical/HITs/Truncation/Properties.agda

```diff
@@ -204,3 +217,174 @@ groupoidTrunc≃Trunc1 =
       (ind (λ _ → squash₂) ∣_∣₂)
       (ind (λ _ → hLevelPath 4 (isOfHLevel∥∥ 2) _ _) (λ _ → refl))
       (g2TruncElim _ _ (λ _ → hLevelPath 4 squash₂ _ _) (λ _ → refl)))
+
+
+
+---- ∥ Ω A ∥ ₙ ≡ Ω ∥ A ∥ₙ₊₁  ----
+
+  
+  {- Proofs of Theorem 7.3.12. and Corollary 7.3.13. in the HoTT book  -}
```

Comment:
I suspect that this has a more direct cubical proof. Did someone try to prove this at some point? (maybe @ecavallo ?)

Reply:
I wouldn't think so, it's an encode-decode proof and those usually don't leave much room for cubical improvement.